packages/cozy-client/src/models/note_schema.js

```diff
@@ -0,0 +1,526 @@
+// taken from a debug of @atlakit/editor/editor-core/create-editor/create-editor
+// L139 (new Schema({nodes ,marks}))
+// static because the @atlaskit code base requires a real navigator
```

Comment:
What is static ?

Reply:
This was moved from the cozy-notes app. It was meant to explain that the schema was not taken at runtime from prose-mirror (which would have been better, in order to always be up  to date). This is a copy that we will haveto update manually